### PR TITLE
Add proper margins to Content Subheader element

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1108
[Problem] Some elements overlap
[Solution] - Creating proper paddings (according to guide and App reference)

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
I started with testing this PR.
No overlapping occurs but we need to improve spacing according to the guideline
See
image
Vs
image

File: examples/mobile/ScheduleApp/css/style.css [hunk 1]
```diff
@@ -81,4 +81,9 @@
 	position: relative;
 	top: 1px;
 	margin: auto;
+}
+
+.content-subheader {
+	height: 52px;
```
Comment (lslachciak) [inline examples/mobile/ScheduleApp/css/style.css]: this seems to be a too big gap according to the GUI (2.1)
image
and 
Reference app 
f/[WebApp]Basic_Reference_GUI_V1.0_200311/index.html#artboard9
image

I wander if we should do the styling here - shouldn't it be a TAU library part?

Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Done!

**Review by lslachciak — APPROVED** [review]
works fine, looks good

**Pull request merged**